src/pages/index.js

```diff
@@ -13,111 +14,259 @@ import {
   validationSettings,
   userNameSelector,
   userDescriptionSelector,
-  imageModalSelector,
+  avatarModal,
   profileModalSelector,
   cardModalSelector,
   cardListSelector,
   profileEditButton,
   modalNameInput,
   modalDescriptionInput,
   addNewCardButton,
+  avatarEditButton,
+  avatarSelector,
+  modalChangeProfileAvatarSelector,
+  editModalFormSelector,
+  addCardFormSelector,
+  avatarModalFormSelector,
+  cardModalDelete,
 } from "../utils/constants.js";
+import PopupWithConfirmation from "../components/PopupWithConfirmation.js";
 
-/**************
- * VALIDATION *
- **************/
-const addFormValidator = new FormValidator(
-  validationSettings,
-  addCardEditForm
-);
-addFormValidator.enableValidation();
-
-const editFormValidator = new FormValidator(
-  validationSettings,
-  profileEditForm
-);
-
-editFormValidator.enableValidation();
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/group-12",
+  headers: {
+    authorization: "9c860865-e6d3-4014-b437-60037dde85fb",
+    "Content-Type": "application/json",
+  },
+});
 
 /*************
- * CARD INFO *
+ * USER INFO *
  *************/
+const userInfo = new UserInfo({
+  userNameSelector,
+  userDescriptionSelector,
+  avatarSelector,
+});
+
+let userId, cardSection;
+
+function renderCard(cardData) {
+  const cardImage = createCard(cardData);
+  cardSection.prependItem(cardImage);
+}
 
-const userInfo = new UserInfo({ userNameSelector, userDescriptionSelector });
+Promise.all([api.getUserInfo(), api.getInitialCards()])
+  .then(([userData, cardData]) => {
+    userInfo.setUserInfo({
+      title: userData.name,
+      description: userData.about,
+    });
+
```

Comment:
You should also set user's avatar here using the corresponding method of `userInfo` class